src/lib/ibcclient.ts

```diff
@@ -801,6 +836,7 @@ export class IbcClient {
     connectionId: string,
     version: string
   ): Promise<CreateChannelResult> {
+    this.logger.verbose(`Channel open init: ${portId} => ${remotePortId}`);
```

Comment:
Maybe log the newly created channelID with `this.logger.debug` call right before the return?